docs/cap_anndata_schema.md

```diff
@@ -1370,12 +1370,12 @@ Key-value pair in the `uns` dictionary
 Python dictionary within the `uns` dictionary, with the key the string `[cellannotation_setname]`
 
 
-#### [cellannotation_setname]--metadata
+#### [cellannotation_setname]_metadata
```

Comment:
this should change to `cellannotation_metadata` right?
```suggestion
#### [cellannotation_metadata]_metadata
```

Reply:
@ubyndr it should be [cellannotation_metadata]